jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/event/QuartzJobStartEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.quartzflowui.event;
+
+import org.quartz.JobDetail;
+import org.springframework.context.ApplicationEvent;
+
+public class QuartzJobStartEvent extends ApplicationEvent {
```

Comment:
Lets mention the 'execution' to be more clear - `QuartzJobStartExecutionEvent`.
And the second one - `QuartzJobFinishExecutionEvent`.
Also change names of `'on...Event'` methods related to these events in `JobModelListView` according to new names.

Reply:
changed